services/issuers/index.js

```diff
@@ -0,0 +1,9 @@
+const defaultUrl = "https://govtech.github.io/exampleIssuers.json";
+
+export default function fetchIssuers(url = defaultUrl) {
+  return window
+    .fetch(url)
```

Comment:
This is throwing `'Access-Control-Allow-Origin' header is present on the requested resource.`, not sure if there are settings to be made on dev env to overwrite this.